src/os/bluestore/bluestore_types.h

```diff
@@ -253,6 +253,7 @@ struct bluestore_blob_t {
 
   vector<bluestore_pextent_t> extents; ///< raw data position on device
   uint32_t length;                 ///< logical (decompressed) length
+  uint32_t clength;                ///< compressed length if any
```

Comment:
>compressed_length? No real need to be concise here, I think.
We need that for:
- decrement compressed counter on blob release. I.e. to be able to track real compressed size.
- avoid reading the whole blob for small compressed block